src/api/form/PDFForm.ts

```diff
@@ -591,11 +590,25 @@ export default class PDFForm {
         page.pushOperators(...operators);
       }
 
-      this.acroForm.removeField(field.ref);
-      this.doc.context.delete(field.ref);
+      this.removeField(field);
```

Comment:
I wonder if we should also remove the widgets themselves from their page's `Annots` array after flattening them? They don't seem to impact how readers render the document, but they are dead weight. Not strictly necessary though.